### ISSUE TITLE
Monocular depth estimation

### ISSUE THREAD

**Issue by sdpscnc** [nianticlabs/monodepth2#167]
Hi, thanks for the amazing work @mrharicot and team!
I used KITTI's very small data of about a few hundred frames to verify that the program can start training for my next work. I have not made major changes to the program, except to comment out the depth_gt part. But when I started training, the loss value never dropped, but fluctuated around 0.2. I changed the batch_size and learning rate, but the loss value did not change. Could you please guide me where may there be a problem? Thank you very much!

**Comment by mrharicot** [comment]
Hi,
Did you change anything other than the number of training images?
I have observed that monodepth2 can fail to train on very little data

**Comment by sdpscnc** [comment]
Thanks for your answer!
I have not changed anything else, except to use very little data and comment out the depth_gt part.

**Comment by mrharicot** [comment]
Then it's very likely coming from the lack of training data.

**Comment by sdpscnc** [comment]
I'm sorry, I'm very stupid! I have made a problem in using KITTI data. My data is not same to 2011_09_26_drive_0001_sync. Now, I can train the program with little data and can begin my next work. Thanks for your amazing work!

**Comment by sdpscnc** [comment]
In addtion, whether the training data is continuous frame image or not?

**Comment by mrharicot** [comment]
Hi,
Glad to hear you got it to work!
KITTI has continuous frame data at 10Hz, it is important that there is enough camera motion between consecutive frames to obtain good depth results.
